_includes/pbc/all/install-features/202212.0/install-the-dynamic-multiple-stores.md

```diff
@@ -2518,6 +2518,95 @@ Also you need to change domain name for all endpoints in EU region.
 Please, check `deploy.dev.dynamic-store.yml` file for more details.
 
 
+## Data migration 
+
+You don't need to migrate data in the database for the dynamic store feature. Only changes in the search and storage engine are necessary.
```

Comment:
is it true that we don't need to "migrate data"? Not a comment, just a question

Reply:
We don't have big changes at the base level. All relations used store_id. these relations were present before. Added a few tables for save currency, locale, country in a separate table. And it's all